tools/vpc-flowlogs-enforcer/terraform/templates/cloud_function/main.py

```diff
@@ -0,0 +1,174 @@
+"""
+ Copyright 2020 Google LLC. This software is provided as-is, without warranty
+ or representation for any use or purpose. Your use of it is subject to your
+ agreement with Google.
+"""
+import logging
+import traceback
+import base64
+import json
+import re
+import os
+import sys
+import time
+from datetime import datetime, timezone
+from dateutil import parser
+from googleapiclient import discovery
+
+
+def enable_net_logs(event, context):
```

Comment:
If you restructure the module into smaller functions, this function then will just hold the core logic and should imho be identified as the main entry point, by renaming it to `main`